charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -15,8 +15,8 @@ data:
   config: |-
     # https://prometheus.io/docs/prometheus/latest/configuration/configuration/
     global:
-      scrape_interval: 30s
-      evaluation_interval: 30s
+      scrape_interval: {{ .Values.global.scrape_interval }}
```

Comment:
Just to circle back on what we talked about on zoom, this "global" is inside the prom config, not a helm namespace. We should probably go with `.Values.prometheus.scrape_interval` for this. Maybe for the ones below we can have EG `.Values.prometheus.velero.scrape_interval` and `.Values.prometheus.velero.scrape_timeout`. Just a guess, there might be an existing data structure that makes more sense.

Reply:
the above values configured inside prometheus and inside configmap it has global config so the placeholder is correct. 

when user make a overrides then --set prometheus.global.scrape_interval

Reply:
Un-knowingly i too worked in this ticket..
What @danielhoherd is pointing does makes sense to me, IMO there is no need to mention global values for the local prometheus charts, We should override the values from the main values.yaml in the basedir and use --set flag to replace..
https://github.com/astronomer/astronomer/pull/1578

Reply:
Oh dang, yeah, this is basically the same thing we were talking about in https://github.com/astronomer/issues/issues/4782

Reply:
This is in a subchart (prometheus), so when the user is installing the astronomer (main) chart, they will still need to scope to this namespace (eg `--set prometheus.velero.scrape_interval=30s`) right? If we were to template it as `.Values.prometheus.velero.scrape_interval`, wont the user have to use `--set prometheus.prometheus.velero.scrape_interval=30s`? I would much prefer the first option.

Reply:
i much prefer not using global namespace but setting all the values in the prometheus subchart values.yaml for documentation and not using the global astronomer/astronomer key unless absolutely necessary as Daniel mentioned [here](https://github.com/astronomer/astronomer/pull/1495#discussion_r866455077)

Reply:
@stuart23 How it works is, if you have provided the values in the main values yaml, https://github.com/astronomer/astronomer/blob/master/values.yaml
this will override the local prometheus chart values otherwise default values are picked up from local chart, if you don't give any values